app/store/ephemeral_store.ts

```diff
@@ -146,6 +148,14 @@ class EphemeralStore {
     isAddingToTeam = (teamId: string) => {
         return this.addingTeam.has(teamId);
     };
+
+    setPushProxyVerificationState = (serverUrl: string, state: string) => {
+        this.pushProxyVerification[serverUrl] = state;
+    };
+
+    getPuhsProxyVerificationState = (serverUrl: string) => {
```

Comment:
Optional - small typo.
```suggestion
    getPushProxyVerificationState = (serverUrl: string) => {
```